swagger_spec_validator/validator20.py

```diff
@@ -21,5 +22,101 @@ def validate_spec_url(spec_url):
 
 
 def validate_spec(spec_json):
+    """Validates a Swagger 2.0 API Specification given a json dict.
```

Comment:
Ok. I will also add a `:type spec_json: dict` to make it clear that the input is a dict and not a json string.